pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -967,6 +975,14 @@ func (m *kubeGenericRuntimeManager) removeContainerLog(ctx context.Context, cont
 	}
 	// Remove the legacy container log symlink.
 	// TODO(random-liu): Remove this after cluster logging supports CRI container log path.
```

Comment:
this TODO is now out of place

Reply:
If I understand correctly, this TODO is means when cluster logging supports container logs defined in CRI, then don't need support legacy container log? for compatibility?

I don't understand how cluster logging is done now, but this PR does not change the `container log`, but only changes the `termination log`. Should this TODO be retained?